daemon/logger/syslog/syslog.go

```diff
@@ -167,3 +178,31 @@ func parseFacility(facility string) (syslog.Priority, error) {
 
 	return syslog.Priority(0), errors.New("invalid syslog facility")
 }
+
+func parseTag(ctx logger.Context) (string, error) {
+	tag := ctx.Config["syslog-tag"]
```

Comment:
Yes, something along that line. I'm not really a code reviewer here :blush:, but I expect that other log-drivers may want to use templating as well, so having something more reusable would be great.

Perhaps some other people have suggestions here (@LK4D4 ?)